frontend/src/components/common/SideNav.vue

```diff
@@ -52,10 +54,10 @@ const props = defineProps({
                             @click="router.push(child.link)"
                         >
                             <Icon
-                                class="p-sidenav-color-icon p-sidenav-padding-icon"
-                                small
+                                class="custom-carbon-icon--sidenav"
```

Comment:
oh wait, since you're passing the class as a parameter to the Icon component, you need to use ":class", otherwise the Icon component can't get it